README.md

```diff
@@ -299,6 +300,14 @@ If you anticipate having large job collections (ie. with over 1000 jobs at a tim
 doing custom queires on the database, you will want to create appropriate additional indexes to
 ensure that your application performs well.
 
+### Removing Old Jobs
+
+Unless you do something to prevent it, completed and removed jobs will accumulate in your database. In some scenarios this may be desirable, but if it's not, there are a few options to clean out old jobs from the database:
```

Comment:
> ... completed and *removed* jobs ...
 
Do you mean *cancelled* jobs?  I guess failed jobs will also build up, but some other mechanism will presumably need to handle those...